src/CitizenProject/CitizenProjectManager.php

```diff
@@ -481,9 +482,22 @@ public function approveCommitteeSupport(Committee $committee, CitizenProject $ci
         }
 
         $committeeSupport->approve();
+        $this->getManager()->persist($committeeSupport);
```

Comment:
There always a cascade persist set, you should call `$this->citizenProject->addCommitteeSupport($this)` in the support constructor instead.